agent/base.ut.post

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; sh-indentation: 4; tab-width: 8 -*-
+
+# base.ut: contains the overrides of the base functions and environment
+#          variables needed for the unit test environments.
+
+function timestamp {
+    echo "1900-01-01T00:00:00.000000"
+}
+
+export date="1900-01-01T00:00:00"
+export date_suffix="1900.01.01T00.00.00"
+
+function check_enable_copr {
+    # For the unit test environment, we only echo the first arguemnt passed to
```

Comment:
I hate to start an "arguemnt" ... but the word here is misspelled.

The revised comment, however, has the marked advantage of being actually useful! Thanks.